libs/components/datetime/src/lib/modules/date-range-picker/date-range-picker.component.ts

```diff
@@ -476,44 +484,28 @@ export class SkyDateRangePickerComponent
     // For example, if we change the value of the form group internally, we don't want the event
     // listeners to be triggered, as those are reserved for user interactions.
```

Comment:
Does this comment still apply now that we're removing `emitEvent: false`?

Reply:
Good catch - removed